### ISSUE TITLE
Add tokio-tracing

### ISSUE THREAD

**Issue by bikeshedder** [bikeshedder/deadpool#174]
This is related to #67

**Comment by elpiel** [comment]
Yes! That's actually a great start, thank you! I was looking into implementing Debug as well but you were very fast to do it 😅

Here's also an idea - rather than not showing the fields that don't have Debug, we can just use a string representation of the type?

**Comment by HigherOrderLogic** [comment]
Any update on this?

**Comment by elpiel** [comment]
@HigherOrderLogic it should be possible to replace `log` with `tracing` (https://docs.rs/tracing) in the necessary crates.
E.g. for `deadpool-postgres` there are exactly 3 places that need to be replaces:

https://github.com/bikeshedder/deadpool/blob/ddd9eafa39e0c823315c3b2bff3767abafa0b05f/postgres/src/lib.rs

**Comment by bikeshedder** [comment]
I'm a `tracing`-noob so to say. What tracing/logging messages would be needed in deadpool to be more useable?

**Comment by bikeshedder** [comment]
I'm currently in process of a [`0.10`](https://github.com/bikeshedder/deadpool/milestone/4) release and if replacing `log` by `tracing` solves 90% of the pain I'm more than happy to include it in this release: https://github.com/bikeshedder/deadpool/tree/milestone/v0.10

**Comment by elpiel** [comment]
A good starting point IMO is to just replace `log` with `tracing` everywhere.
I don't know what new logging messages would be beneficial though.
